buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1050,12 +1044,11 @@ class BeamModulePlugin implements Plugin<Project> {
       }
       project.check.dependsOn project.javadoc
 
-      // Apply the eclipse and apt-eclipse plugins.  This adds the "eclipse" task and
+      // Apply the eclipse plugis.  This adds the "eclipse" task and
```

Comment:
s/plugis/plugins

(could be fixed in a follow up)

Reply:
Done.